src/EntityEmbedDisplay/EntityEmbedDisplayManager.php

```diff
@@ -98,4 +98,24 @@ public function getDefinitionOptionsForEntity(EntityInterface $entity) {
     }, $definitions);
   }
 
+  /**
+   * Provides a list of plugins that can be used for a certain entity type.
+   *
+   * @param string $entity_type
+   *   The entity type id.
+   *
+   * @return array
+   *   An array of valid plugin labels, keyed by plugin ID.
+   */
+  public function getDefinitionOptionsForEntityType($entity_type) {
+    $definitions = $this->getDefinitions();
+    $valid_definitions = array();
+    foreach ($definitions as $id => $definition) {
+      if ($definition['entity_types'] == FALSE || in_array($entity_type, $definition['entity_types'])) {
+        $valid_definitions[$id] = $definition['label'];
```

Comment:
No really. Since this function will mostly be used to populate options in form, its better to just return the label otherwise the caller will have to extract the label from the definition.

FWIW, this behavior is similar to the existing `getDefinitionOptionsForEntity()` method.